ts/features/zendesk/navigation/navigator.tsx

```diff
@@ -0,0 +1,22 @@
+import { createStackNavigator } from "react-navigation-stack";
+import ZendeskSupportHelpCenter from "../screens/ZendeskSupportHelpCenter";
+import ZendeskAskPermissions from "../screens/ZendeskAskPermissions";
+import ZendeskChooseCategory from "../screens/ZendeskChooseCategory";
+import { zendeskEnabled } from "../../../config";
+import ZENDESK_ROUTES from "./routes";
+
+const zendeskConfigMap = zendeskEnabled
+  ? {
+      [ZENDESK_ROUTES.HELP_CENTER]: { screen: ZendeskSupportHelpCenter },
+      [ZENDESK_ROUTES.ASK_PERMISSIONS]: { screen: ZendeskAskPermissions },
+      [ZENDESK_ROUTES.CHOOSE_CATEGORY]: { screen: ZendeskChooseCategory }
+    }
+  : {};
+
+export const zendeskSupportNavigator = createStackNavigator(zendeskConfigMap, {
+  // Let each screen handles the header and navigation
+  headerMode: "none",
+  defaultNavigationOptions: {
+    gesturesEnabled: false
+  }
+});
```

Comment:
It doesn't work in this way.
If you put `zendeskEnabled===false` the app doesn't start

<img src="https://user-images.githubusercontent.com/822471/144573145-d7d9de7e-e682-488c-ae70-c723ca02b78a.png" height="600"/>

I suggest to apply the same patter [used here](https://github.com/pagopa/io-app/blob/1c2d2b54b96ad5b075e702857f8517b1ea06759d/ts/navigation/WalletNavigator.ts#L222) for example
I tested this code by editing the file `ts/navigation/AppNavigator.ts`

```typescript
const configMap = {
  [ROUTES.INGRESS]: {
    // This is the first screen that gets loaded by the app navigator
    // On component mount, the screen will dispatch an
    // APPLICATION_INITIALIZED action that gets handled by the startup saga.
    screen: IngressScreen
  },
  [ROUTES.BACKGROUND]: {
    screen: BackgroundScreen
  },
  [ROUTES.AUTHENTICATION]: {
    // The navigator used during authentication
    screen: AuthenticationNavigator
  },
  [ROUTES.ONBOARDING]: {
    // The navigator user during onboarding for authenticated users
    screen: OnboardingNavigator
  },
  [ROUTES.MAIN]: {
    // The navigator used for authenticated users on onboarding completed
    screen: MainNavigator
  },
  [ROUTES.WORKUNIT_GENERIC_FAILURE]: {
    screen: WorkunitGenericFailure
  }
};

const zendeskMap = zendeskEnabled
  ? {
      [ZENDESK_ROUTES.HELP_CENTER]: {
        screen: zendeskSupportNavigator
      }
    }
  : {};

/**
 * The main stack of screens of the Application.
 */
const navigator = createStackNavigator(
  { ...configMap, ...zendeskMap },
  {
    // Let each screen handle the header and navigation
    headerMode: "none",
    defaultNavigationOptions: {
      gesturesEnabled: false
    }
  }
);

export default navigator;
```

Reply:
Thank you! Critical! 
Fixed in [71ff382](https://github.com/pagopa/io-app/pull/3538/commits/71ff38266f4599e0cec703a570d6499eecfa5e5c)